aptos-move/move-examples/account_abstraction/bls12381_single_key/sources/bls12381_single_key.move

```diff
@@ -1,4 +1,4 @@
-module aa::single_key {
+module aa::bls12381_single_key {
```

Comment:
Let's put a doc comment at the top of these explaining the examples.

We've done a bad job of this in the past.

For example this one might be:
```
/// An example showing how to use Account Abstraction to use a BLS12381 key to sign
/// transactions instead of a natively supported signing key on Aptos.
```